src/coriolis.jl

```diff
@@ -69,14 +69,16 @@ end
     T(0.5) * f * (avgx_f2c(grid, u, i,  j-1, k) + avgx_f2c(grid, u, i, j, k))
 
 @inline fv(i, j, k, grid::RegularCartesianGrid{T}, f₀, β, v) where T =
-    T(0.5) * (f₀ + β * grid.yC[j]) * (avgy_f2c(grid, v, i-1,  j, k) + avgy_f2c(grid, v, i, j, k))
+     T(0.5) * (f₀ + β * @inbounds(grid.yC[j])) * (avgy_f2c(grid, v, i-1,  j, k) + avgy_f2c(grid, v, i, j, k))
```

Comment:
You can put `@inbounds` at the start of the expression and every array indexing operation will elide bounds checking:
```julia
@inbounds @inline fv(i, j, k, grid::RegularCartesianGrid{T}, f₀, β, v) where T =
    T(0.5) * (f₀ + β * grid.yC[j]) * (avgy_f2c(grid, v, i-1,  j, k) + avgy_f2c(grid, v, i, j, k))
```